python/cuml/dask/datasets/blobs.py

```diff
@@ -80,17 +80,20 @@ def make_blobs(nrows, ncols, centers=8, n_parts=None, cluster_std=1.0,
     Parameters
     ----------
 
-    nrows : int
+    n_samples : int
         number of rows
-    ncols : int
+    n_features : int
         number of features
-    n_centers : int (default = 8)
-        number of centers to generate
+    centers : int or array of shape [n_centers, n_features],
```

Comment:
I know you're matching sklearn, but I don't know why they don't just set `centers=3` as a default param?

Reply:
I was thinking the same thing. I'm actually not sure why it was ever set to `8` in cuML either. I figured I'd follow their parameters as closely as possible.